docs/guides/openstack_guides/putty.md

```diff
@@ -1,72 +1,45 @@
-# SSH for Windows using putty 
-If you're using an older version of Windows, you might not have access to the OpenSSH client. Instead, people often use the tool [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html). 
-
-
-## Convert key
-With the native OpenSSH client you can use this private key(pem-file) directly. For Putty you have to convert your key first, by completing the following steps:
-
-
-
-
-* Start PuTTYgen (e.g., from the Start menu, click All Programs > PuTTY > PuTTYgen).
-* Click Load
-
-![Alt Description](../img/openstack/puttykeygenerator.png?raw=true "Title")
-
-* Browse to the location of the private key file that you want to convert (e.g., MyPersonalKey.pem).
-
-Note: By default, PuTTYgen displays only files with extension .ppk; you'll need to change that to display files of all types in order to see your .pem key file. The private key file must end with a newline character or PuTTYgen cannot load it correctly.
-
-![Alt Description](../img/openstack/puttysavekey.png?raw=true "Title")
-
+# Connect to instance from Windows (using PuTTY) 
+Some versions of Windows do not support OpenSSH out of the box. This means that you can't use the SSH-key generated by the Openstack-platform (in the format *.pem). We recommend a solution where you use [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) to convert your SSH-key to a Windows supported format and then connect to your instance. 
```

Comment:
Isn't it more of a question of having the key in a PuTTY-supported format than a Windows-supported format? If so, this "Windows format" term is mentioned once more further down the page.

Reply:
Det kan du sådan set have ret i.